VmAgent.Core/Interfaces/SessionHostContainerConfiguration.cs

```diff
@@ -46,7 +58,14 @@ protected override string GetLogFolder(string logFolderId, VmConfiguration vmCon
         {
             // The VM host folder corresponding to the logFolderId gets mounted under this path for each container.
             // So the logFolderId itself isn't of much significance within the container.
-            return vmConfiguration.VmDirectories.GameLogsRootFolderContainer + Path.DirectorySeparatorChar;
+            if (_sessionHostManager != null && _sessionHostManager.LinuxContainersOnWindows)
```

Comment:
maybe you can do

 if(_sessionHostManager?.LinuxContainerOnWindows) ???

Reply:
Thanks for the feedback! I think I can do **_sessionHostManager?.LinuxContainersOnWindows ?? false** as LinuxContainerOnWindows is bool type variable? let me know if there is another idea

Reply:
oh yeah, you're right, sorryz!